File: Tarea7/Banco.java
import java.util.ArrayList;

import java.util.Scanner;
public class Banco {
    private ArrayList<Empleado> empleados;
    public Banco() {
        empleados = new ArrayList<>();
    }
    public void agregarEmpleado(Empleado empleado) {
        empleados.add(empleado);
    }
    public void mostrarEmpleados() {
        System.out.println("Estos son los empleados registrados");
        for(Empleado empleado : empleados) {
            System.out.println(" ");
            System.out.println("Nombre del empleado: " + empleado.getNombre());
            System.out.println("Edad del empleado: " + empleado.getEdad());
            System.out.println("Salario registrado: " + empleado.getSalario());
            System.out.println("Numero de tu cuenta: " + empleado.getCuenta().getNumeroCuenta());
            System.out.println("Tipo de cuenta seleccionado: " + empleado.getCuenta().getTipoCuenta());
            System.out.println(" ");
        }
    }
    public void mostrarEmpleado() {
        System.out.println("Cuentas registradas: ");
        for(int x = 0 ; x < empleados.size() ; x++) {
            System.out.println(x + ". " + empleados.get(x).getNombre());
        }
        System.out.println("Empleado que quiera ver: ");
        Scanner scanner = new Scanner(System.in);
        int numEmpleado = scanner.nextInt();
        if (numEmpleado >= 0 && numEmpleado < empleados.size()) {
            System.out.println(" ");
            System.out.println("Los datos del empleado " + numEmpleado + " son: ");
            System.out.println("Nombre del titular : " + empleados.get(numEmpleado).getNombre());
            System.out.println("Edad del empleado: " + empleados.get(numEmpleado).getEdad());
            System.out.println("Salario registrado: " + empleados.get(numEmpleado).getSalario());
            System.out.println("Numero de cuenta: " + empleados.get(numEmpleado).getCuenta().getNumeroCuenta());
            System.out.println("Tipo de cuenta: " + empleados.get(numEmpleado).getCuenta().getTipoCuenta());
            System.out.println(" ");

        }
    }

    public void elegirCuentaRetiro() {
        System.out.println("Cuentas registradas: ");
        for(int x = 0 ; x < empleados.size() ; x++) {
            System.out.println(x + ". " + empleados.get(x).getNombre() + " con numero de cuenta: "  + empleados.get(x).getCuenta().getNumeroCuenta());
        }
        System.out.println("Elige tu la cuenta para retirar: ");
        Scanner scanner = new Scanner(System.in);
        int cuentaRetiro = scanner.nextInt();
        if(cuentaRetiro >= 0 && cuentaRetiro < empleados.size()) {
            empleados.get(cuentaRetiro).getCuenta().retiro();
        }
    }
    public void elegirCuentaDeposito() {
        System.out.println("Cuentas registradas: ");
        for(int x = 0 ; x < empleados.size() ; x++) {
            System.out.println(x + ". " + empleados.get(x).getNombre() + " con numero de cuenta: "  + empleados.get(x).getCuenta().getNumeroCuenta());
        }
        System.out.println("Elige tu la cuenta para depositar: ");
        Scanner scanner = new Scanner(System.in);
        int cuentaDeposito = scanner.nextInt();
        if(cuentaDeposito >= 0 && cuentaDeposito < empleados.size()) {
            empleados.get(cuentaDeposito).getCuenta().deposito();
        }
    }
    public void verSaldoCuenta() {
        System.out.println("Cuentas registradas: ");
        for(int x = 0 ; x < empleados.size() ; x++) {
            System.out.println(x + ". " + empleados.get(x).getNombre()+ " con numero de cuenta: "  + empleados.get(x).getCuenta().getNumeroCuenta());
        }
        System.out.println("Elige tu la cuenta para retirar: ");
        Scanner scanner = new Scanner(System.in);
        int cuentaSaldo = scanner.nextInt();
        if (cuentaSaldo >= 0 && cuentaSaldo < empleados.size()) {
            empleados.get(cuentaSaldo).getCuenta().getSaldoCuenta();
        }
    }
}

File: Tarea7Python/Banco.py
class Banco:
    def __init__(self):
        self.empleados = []

    def agregar_empleado(self, empleado):
        self.empleados.append(empleado)

    def mostrar_empleados(self):
        print("Estos son los empleados registrados")
        for empleado in self.empleados:
            print(" ")
            print("Nombre del empleado: " + empleado.get_nombre())
            print("Edad del empleado: " + str(empleado.get_edad()))
            print("Salario registrado: " + str(empleado.get_salario()))
            print("Numero de tu cuenta: " + empleado.get_cuenta().get_numero_cuenta())
            print("Tipo de cuenta seleccionado: " + empleado.get_cuenta().get_tipo_cuenta())
            print(" ")

    def mostrar_empleado(self):
        print("Cuentas registradas: ")
        for x in range(len(self.empleados)):
            print(str(x) + ". " + self.empleados[x].get_nombre())
        print("Empleado que quiera ver: ")
        num_empleado = int(input())
        if 0 <= num_empleado < len(self.empleados):
            print(" ")
            print("Los datos del empleado " + str(num_empleado) + " son: ")
            print("Nombre del titular : " + self.empleados[num_empleado].get_nombre())
            print("Edad del empleado: " + str(self.empleados[num_empleado].get_edad()))
            print("Salario registrado: " + str(self.empleados[num_empleado].get_salario()))
            print("Numero de cuenta: " + self.empleados[num_empleado].get_cuenta().get_numero_cuenta())
            print("Tipo de cuenta: " + self.empleados[num_empleado].get_cuenta().get_tipo_cuenta())
            print(" ")

    def elegir_cuenta_retiro(self):
        print("Cuentas registradas: ")
        for x in range(len(self.empleados)):
            print(str(x) + ". " + self.empleados[x].get_nombre() + " con numero de cuenta: " +
                  self.empleados[x].get_cuenta().get_numero_cuenta())
        print("Elige tu la cuenta para retirar: ")
        cuenta_retiro = int(input())
        if 0 <= cuenta_retiro < len(self.empleados):
            self.empleados[cuenta_retiro].get_cuenta().retiro()

    def elegir_cuenta_deposito(self):
        print("Cuentas registradas: ")
        for x in range(len(self.empleados)):
            print(str(x) + ". " + self.empleados[x].get_nombre() + " con numero de cuenta: " +
                  self.empleados[x].get_cuenta().get_numero_cuenta())
        print("Elige tu la cuenta para depositar: ")
        cuenta_deposito = int(input())
        if 0 <= cuenta_deposito < len(self.empleados):
            self.empleados[cuenta_deposito].get_cuenta().deposito()

    def ver_saldo_cuenta(self):
        print("Cuentas registradas: ")
        for x in range(len(self.empleados)):
            print(str(x) + ". " + self.empleados[x].get_nombre() + " con numero de cuenta: " +
                  self.empleados[x].get_cuenta().get_numero_cuenta())
        print("Elige tu la cuenta para retirar: ")
        cuenta_saldo = int(input())
        if 0 <= cuenta_saldo < len(self.empleados):
            self.empleados[cuenta_saldo].get_cuenta().get_saldo_cuenta()
